example-plugins/homebridge-samplePlatform/index.js

```diff
@@ -0,0 +1,197 @@
+var http = require('http');
+var Accessory, Service, Characteristic, UUIDGen;
+
+module.exports = function(homebridge) {
+  console.log("homebridge API version: " + homebridge.version);
+
+  // Accessory must be created from PlatformAccessory Constructor
+  Accessory = homebridge.platformAccessory;
+
+  // Service and Characteristic are from hap-nodejs
+  Service = homebridge.hap.Service;
+  Characteristic = homebridge.hap.Characteristic;
+  UUIDGen = homebridge.hap.uuid;
+  
+  // For platform plugin to be considered as dynamic platform plugin,
+  // registerPlatform(pluginName, platformName, constructor, dynamic), dynamic must be true
+  homebridge.registerPlatform("homebridge-samplePlatform", "SamplePlatform", SamplePlatform, true);
+}
+
+// Platform constructor
+// config may be null
+// api may be null if launched from old homebridge version
+function SamplePlatform(log, config, api) {
+  console.log("SamplePlatform Init");
+  this.log = log;
+  this.config = config;
+  this.accessories = [];
+
+  this.requestServer = http.createServer(function(request, response) {
+    if (request.url === "/add") {
+      this.addAccessory();
+      response.writeHead(204);
+      response.end();
+    }
+
+    if (request.url == "/remove") {
+      this.removeAccessory();
+      response.writeHead(204);
+      response.end();
+    }
+  }.bind(this));
+
+  this.requestServer.listen(18081, function() {
+    console.log("Server Listening...");
+  });
+
+  if (api) {
+      // Save the API object as plugin needs to register new accessory via this object.
+      this.api = api;
+
+      // Listen to event "didFinishLaunching", this means homebridge already finished loading cached accessories
+      // Platform Plugin should only register new accessory that doesn't exist in homebridge after this event.
+      // Or start discover new accessories
+      this.api.on('didFinishLaunching', function() {
+        console.log("Plugin - DidFinishLaunching");
+      }.bind(this));
+  }
+}
+
+// Function invoked when homebridge tries to restore cached accessory
+// Developer can configure accessory at here (like setup event handler)
+// Update current value
+SamplePlatform.prototype.configureAccessory = function(accessory) {
+  console.log("Plugin - Configure Accessory: " + accessory.displayName);
+
+  // set the accessory to reachable if plugin can currently process the accessory
+  // otherwise set to false and update the reachability later by invoking 
+  // accessory.updateReachability()
+  accessory.reachable = true;
```

Comment:
I tried to set that to `false` and it had no effect on my test device (iOS 9.3b2, EVE app)